.circleci/config.yml

```diff
@@ -203,6 +203,59 @@ jobs:
           command: |
             python bump_bot_team.py run_pr_update
 
+  run_versions_update:
+    working_directory: ~/repo
+    docker:
+      - image: continuumio/miniconda3:latest
+    steps:
+      - checkout
+      - run:
+          name: Update Conda
+          command: |
+            conda config --set always_yes True
+            conda config --prepend channels conda-forge
+            conda update --all
+            conda install --quiet --yes --file requirements.txt
+      - run:
+          name: Git user
+          command: |
+            git config --global user.name regro-cf-autotick-bot
+            git config --global user.email circleci@cf-graph.regro.github.com
+      - run:
+          name: Clone and Setup
+          command: |
+            export START_TIME=$(date +%s)
+            export TIMEOUT=7200
+            set -e
+            git clone https://github.com/regro/cf-scripts.git
+            cd cf-scripts
+            # remove/comment to run on master
+            # git fetch origin --verbose
+            # git checkout origin/v1
+            export GIT_FULL_HASH=$(git rev-parse HEAD)
+            conda create -n run_env --quiet --file requirements/run
+            source activate run_env
+            conda info
+            conda config --show-sources
+            conda list --show-channel-urls
+            python setup.py develop
+            cd ..
+            git clone --depth=1 https://github.com/regro/cf-graph-countyfair.git cf-graph
+            git clone --depth=1 https://github.com/conda-forge/conda-forge-pinning-feedstock.git
+      - run:
+          name: populate versions
+          command: |
+            cd cf-graph
```

Comment:
I think it is not necessary to change, but I think you can use
``working_directory: ~/cf-graph``

Reply:
It's a good idea but i need the working directory as `~/regro` to be able to go to `cf-graph-county fair` and run the update upstream there right ?